hpy/universal/src/api.c

```diff
@@ -121,6 +121,29 @@ ctx_CallRealFunctionFromTrampoline(HPyContext ctx, struct _object *self,
     }
 }
 
+typedef HPy (*HPyMeth_Keywords)(HPyContext, HPy self, HPy *args, HPy_ssize_t,
+                                HPy kw);
+
+static struct _object *
+ctx_CallRealFunctionWithKeywordsFromTrampoline(HPyContext ctx,
```

Comment:
strictly speaking this is not wrong. However, it is possible to implement the same functionality by adding a case to the existing `CallRealFunctionFromTrampoline`, by adding a `*kw` param which will be `NULL` in the non-keyword cases

Reply:
Fixed!